homeassistant/components/androidtv/strings.json

```diff
@@ -32,12 +32,12 @@
         "title": "Android TV Options",
         "data": {
           "apps": "Configure applications list",
-          "get_sources": "Whether or not to retrieve the running apps as the list of sources",
-          "exclude_unnamed_apps": "Exclude app with unknown name",
-          "screencap": "Determines if album art should be pulled from what is shown on screen",
+          "get_sources": "Use the running apps as the list of sources",
```

Comment:
What `get_sources` really means is "Retrieve the list of running apps." There was an issue a while back that caused the ADB connection to get broken frequently, and I added this option in order to mitigate that. It probably didn't make a difference, but the real issue has since been fixed. Anyways...

If `get_sources` is true, the running apps will be retrieved and used as the sources. If it is false, there will be only one source: the current app.

Reply:
So you suggest to replace here with Retrieve? I could use your explanation for documentation that I'm working on...